scripts/remove

```diff
@@ -28,8 +27,5 @@ sudo rm -f /etc/systemd/system/$app.service
 # remove database!!!
 sudo mongo rocketchat --eval "db.dropDatabase()"
 
-# remove user!!!
-sudo userdel -r $serviceuser
```

Comment:
Why not remove the user here anymore? It does get added on restore again and I would expect the user to disappear when I remove the app